auction-server/src/auction.rs

```diff
@@ -1249,21 +1265,20 @@ async fn verify_signatures_svm(
                     ),
                 })
                 .await;
-            opportunities.into_iter().any(|opportunity| {
-                let mut missing_signers = opportunity.get_missing_signers();
-                missing_signers.push(*relayer_pubkey);
-                all_signature_exists_svm(&message_bytes, accounts, &signatures, &missing_signers)
-            })
+            let opportunity = opportunities
+                .first()
```

Comment:
i assume you're trying to check the most recent opportunity added? just make sure that first() returns the most recent and not the oldest

Reply:
not really. There should be only one opportunity but the APIs return `Vec`.
@danimhr please correct me if I'm wrong

Reply:
mmm I'm not sure https://github.com/pyth-network/per/blob/main/auction-server/src/opportunity/repository/add_opportunity.rs#L41

Reply:
I added some comments here